### PR TITLE
Add code block in benchcmp output

### PR DESCRIPTION
**Pull request by bradbeam**
### 1. Why is this pull request needed and what does it do?

Adds some formatting to the benchmark results on PR

### CODE REVIEW

File: .benchmark.sh [hunk 1]
```diff
@@ -5,7 +5,7 @@ set -e +o pipefail
 if [ "$TRAVIS_PULL_REQUEST" != "false" ] ; then
     echo -e "NOTE: The CPU benchmarks are performed on Travis VMs and vary widly between runs," > .benchmark.body
     echo -e " you can't trust them. The memory benchmarks are OK\n\n" >> .benchmark.body
-    cat .benchmark.log >> .benchmark.body
+    cat .benchmark.log | awk '/^benchmark old/ { printf "%s\n%s\n", "```", $0 }; /^$/ { print "```" }; /^Bench/ { print $0 }; END{ print "```" }' >  .benchmark.body
```
Comment (miekg) [inline .benchmark.sh]: had the same thought :)

However you need a `>>` instead of `>` because now you're overwritting .benchmark.body ?

Comment (bradbeam) [reply .benchmark.sh]: whoops.

**Comment by miekg** [comment]
it's doing it almost right: https://github.com/coredns/coredns/pull/2137#issuecomment-425136218

**Comment by bradbeam** [comment]
Well that's just odd.  Looks like the headers are gone / missing.  Maybe I just copied old results.

**Comment by bradbeam** [comment]
Yeah that was it. I copied from the comment versus editing and copying there. The whitespace was off.

**Pull request merged**